aptos-move/move-examples/veiled_coin/Move.toml

```diff
@@ -10,3 +10,4 @@ std = "0x1"
 aptos_std = "0x1"
 aptos_framework = "0x1"
 veiled_coin = "_"
+source_addr = "_"
```

Comment:
Same suggestion of using `developer_address`.

Reply:
Replaced with `developer_addr`